packages/apollos-data-prayer/src/prayer-requests/data-source.js

```diff
@@ -155,53 +152,53 @@ export default class PrayerRequest extends RockApolloDataSource {
       .find(id)
       .get();
 
-  getFromIds = (ids) =>
-    this.request().filterOneOf(ids.map((id) => `Id eq ${id}`));
+  getFromIds = async (ids) => {
+    const idsFilter = ids.map((id) => `Id eq ${id}`);
+    return this.request()
+      .filterOneOf(idsFilter)
+      .get();
+  };
```

Comment:
Moving the filter out ... is that just for readability? Also, do we need the `async` if we're not `await`ing anything?

Reply:
Or when this function is called later, does it automatically `await`? I see on line 199 you removed the `await`.

Reply:
done